Sources/Stencil/StringsContext.swift

```diff
@@ -38,42 +38,49 @@ extension StringsFileParser {
       Array(0..<keyPath.count).forEach { _ in keyStructure.removeFirst() }
       let keytail = keyStructure.joined(separator: ".")
 
+      var result: [String: Any] = [
+        "key": entry.key.newlineEscaped,
+        "translation": entry.translation.newlineEscaped,
+        "keytail": keytail
+      ]
+
       if entry.types.count > 0 {
-        let params: [String: Any] = [
-          "types": entry.types.map { $0.rawValue },
+        result["parameters"] = entry.types.map { $0.rawValue }
```

Comment:
I wonder if we shouldn't name this entry `types` or `placeholderTypes` instead of `parameters`. After all, that's more the list of types than a list of parameters. We call if `parameters` because we assume it's gonna be used to derive the parameters of an enum with associated values, but maybe not…